CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+2. Click on "Eclipse ECA", and complete the form.
+
+Be sure to use the same email address in your Eclipse Account that you intend to use when you commit to GitHub.
+
+## How to Contribute
+
+### Discuss
+
+If you feel there is a bug or an issue contribute to discussions in
+[existing issues](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues?q=is%3Aissue+is%3Aopen)
+otherwise [create a new issue](#create-an-issue).
+
+### Create an Issue
+
+If you have identified a bug or just want to share an idea to further enhance
+the project feel free to create a new issue at our projects corresponding 
+[Github Issues page](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/new).
+
+Before doing so please consider searching for potentially suitable 
+[existing issues](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues?q=is%3Aissue+is%3Aopen). 
+
+We also
+use [Github's default label set](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels)
+to classify issues and improve findability.
+
+### Submit a Pull Request
+
+In addition to the contribution guideline made available in the [Eclipse project handbook](https://www.eclipse.org/projects/handbook/#contributing)
+we would appreciate if your pull request applies to the following points:
+
+* Always apply the following copyright header to specific files in your work replacing the fields enclosed by curly brackets "{}" with your own identifying information. (Don't
+  include the curly brackets!) Enclose the text in the appropriate comment syntax for the file format.
+
+    ```text
+    Copyright (c) {year} {owner}[ and others]
+
+    This program and the accompanying materials are made available under the
+    terms of the Apache License, Version 2.0 which is available at
+    https://www.apache.org/licenses/LICENSE-2.0
+
+    SPDX-License-Identifier: Apache-2.0
+
+    Contributors:
+      {name} - {description}
+    ```
+
+* The git commit messages should comply to the following format:
+    ```
+    <component>: <description>
+    ```
+
+  Use the [imperative mood](https://github.com/git/git/blob/master/Documentation/SubmittingPatches)
+  as in "Fix bug" or "Add feature" rather than "Fixed bug" or "Added feature" and
+  [mention the GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
+  e.g. `transfer process: improve logging, closes #3`.
+
+  All committers, and all commits, are bound to
+  the [Developer Certificate of Origin.](https://www.eclipse.org/legal/DCO.php)
+  As such, all parties involved in a contribution must have valid ECAs and commits must
+  include [valid "Signed-off-by" entries](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git). Commits
+  can be signed off by included the `-s` attribute in your commit message, for
+  example, `git commit -s -am 'Interesting Commit Message.`
+
```

Comment:
signed-off is optional, see https://www.eclipse.org/lists/eclipse.org-committers/msg01291.html

```suggestion
  As such, all parties involved in a contribution must have valid ECAs. Additionally commits can
  include a ["Signed-off-by" entry](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git). 
```